assets/components/ovn/single-node/master/daemonset.yaml

```diff
@@ -417,7 +418,7 @@ spec:
             memory: 60Mi
         env:
         - name: OVN_KUBE_LOG_LEVEL
-          value: "4"
+          value: "5"
```

Comment:
nit: why changing this to 5?

Reply:
This was for debugging. Let me change it back.